sys/include/net/gnrc/ipv6.h

```diff
@@ -46,6 +46,13 @@
 extern "C" {
 #endif
 
+
+/**
+ * @defgroup    net_gnrc_ipv6_conf  GNRC IPv6 compile configurations
```

Comment:
so, we agree all configuration groups have the `<dox_module>_conf` name?

Reply:
As long as it is clear, that they have something to do with configuration and are in the `config` group, I don't really care either way.

Reply:
I'm OK with it